Medical Loadout Additions and Changes (Senior Physician Edition)

Eagle0600 opened the pull request:
# Description

Adds a number of medical loadout options I consider missing and makes some other changes, with particular attention to the (relatively) recent addition of the senior physician role.

Additions:
- Generic medical white shoes, as found in their default starting gear and 'drobes.
- Senior physician's labcoat and black shoes, as found in their default starting gear.
- Chemist winter coat and boots
- Paramedic's winter coat, EMT belt, and blue shoes.
- Corpsman's medical HUD, sterile mask, and surgical duffel bag.
- Medical beret and winter coat and boots
- Genetic and virology labcoats and winter coats and boots

Changes:
- Senior physician specific clothing was removed from the uncategorised loadouts and added back to senior physician loadouts, as this was always intended to be temporary.
- Loadout groups were added to medical lingerie, except for paramedic lingerie.
- Generic medical loadouts allowed for medical doctors are now allowed for senior physicians:
  - Medkits
  - Medical labcoat
  - Cybersun surgical cap, scrubs, and windbreaker
- The surgical duffel is now equipment, and will spawn alongside your chosen or default bag rather than replacing it.

Fixes
- Senior physician's empty medical belt loadout no longer requires being a medical doctor.
- Medical doctor's nurse's hat loadout is now no longer erroneously in the uncategorised loadout category.

---

<details><summary><h1>Media</h1></summary>
<p>

![loadoutvirogenetic](https://github.com/user-attachments/assets/50f4e948-4f5a-427d-a337-aea073cbdf7a)
![loadoutseniorphysician](https://github.com/user-attachments/assets/ed5d73d5-2ce3-47f6-b397-2b9448dfe507)
![paramedicloadouts](https://github.com/user-attachments/assets/ccc516e3-4c36-464a-9d55-5bc3d0537136)
![corpsmanloadouts](https://github.com/user-attachments/assets/6844ee8a-839b-41b4-8bfc-c4383059e51c)

</p>
</details>

---

# Changelog

:cl:
- add: Genetic and virology labcoats, winter coats, and winter boots are now available for medical staff.
- add: Chemists can now start with their winter coats and boots.
- add: Senior physicians have a number of new loadout options.
- add: Paramedics can now start with their winter coats and EMT belt.
- add: Corpsmen can now start with a medical HUD and a surgical duffel bag.
- tweak: Doctors can now start with a surgical duffel in addition to their chosen bag instead of replacing it. You will have to re-select the loadout.
- removed: Non-senior medical staff can no longer start with senior physician's uniforms.

Eagle0600 committed:
Revert senior physician clothing moved to uncategorized loadouts
This was temporary and is no longer necessary.
Change restricted to medical to limit surface-area of conflict with M3739's open loadout changes.

Eagle0600 committed:
Senior physician empty medical belt loadout no longer requires being medical doctor.

Eagle0600 committed:
Add loadout groups to medical lingerie

Eagle0600 committed:
Add missing medical loadout items.
Generic medical white shoes.
Senior physician's labcoat and black shoes.

Eagle0600 committed:
Let senior physician start with medkits
With apologies to M3739, this will cause a merge conflict, but should be easily resolved.

Eagle0600 committed:
Add loadout groups to chemical lingerie

Eagle0600 committed:
Put nurse hat in correct category

Eagle0600 committed:
Add more missing medical loadout items
Chemist winter coat and boots
Medical beret and winter coat and boots
Genetic and virology labcoats and winter coats and boots

Eagle0600 committed:
Surgical duffel bag is now equipment.
It will spawn alongside your chosen (or default) bag, without replacing it.

Eagle0600 committed:
Remove duplicate line.
This fixes a critical error introduced in this pull request.

Mnemotechnician commented on Resources/Prototypes/Loadouts/Jobs/Medical/uncategorized.yml:
```diff
@@ -438,6 +422,7 @@
     - !type:CharacterJobRequirement
```
For future reference, generic requirements like this could be replaced with a CharacterDepartmentRequirement to avoid repetition. Not requesting any changes, just a note for the future.

Eagle0600 replied:
In this case, it is specifically excluding the CMO and the medical intern, and in some cases also the chemist. You'll notice that these multi-job requirements were already here, I didn't add them. The decision was a deliberate one.

Mnemotechnician reviewed (approved):
Looks good.

M3739 commented:
Aye. Finally got around to reading this. Once #690 is merged, I'll ring back to give you the green light for those changes you mentioned.

M3739 commented:
#690 was merged, let the fun begin!

Eagle0600 committed:
Merge branch 'Fansana:master' into Senior-Physican-Loadout

Eagle0600 commented:
Alright then.

Eagle0600 committed:
Reverse recent removal of paramedic backpack from paramedic starter gear
The reasons are twofold: First, the existence of the same backpack in the loadouts does not mean it should be default starting gear for a paramedic. Secondly, this change was not made to the matching satchel and duffel, making default paramedic starter gear inconsistent.

Eagle0600 committed:
Add missing paramedic loadouts
Blue shoes and EMT belts from starting equipment.
Winter coat from locker.
The GPS was not added because, for now, it is spawned in pockets by default, and I don't believe there's any way to override that.
Also mark loadouts I forgot to mark as belonging to Floof last time.

Eagle0600 committed:
Rectify paramedic locker contents.
The winter coat and GPS were in the hardsuit locker, but not the basic locker.

Eagle0600 committed:
Add loadout groups to corpsman lingerie.
Also correctly mark the medical security gasmask loadout as added by Floof

Eagle0600 committed:
Add descriptive names to corpsman and paramedic bag loadouts.

Eagle0600 committed:
Add descriptive loadout name for corpsman webbing loadout

Eagle0600 committed:
Add medical HUD and sterile mask to corpsman loadouts.
These items are given to corpsmen either on roundstart or in lockers.
Other items in the locker, such as the handheld GPS and tracker implant, were deliberately excluded.

Eagle0600 committed:
Add surgical duffel bag to corpsman loadouts

Eagle0600 commented:
Made some changes to paramedic and corpsman loadouts, I'll test 'em before pushing.
In summary:
- Add paramedics: Blue shoes, EMT belts, and winter coats.
- Add corpsman: Medical HUD an sterile mask.
- Add a descriptive the webbing loadout.
- Add loadout groups to paramedic and corpsman lingerie.
- Rectified paramedic locker contents.

Eagle0600 committed:
Reverse "Add descriptive names to corpsman and paramedic bag loadouts."
It turns out that these entries already exist

Eagle0600 committed:
Fix silly but critical errors made when adding groups to paramedic lingerie.

Eagle0600 committed:
Place the corpsman's surgical duffel bag loadout in the correct category.

Eagle0600 commented:
The build fail is "Server prototype ComplexXenoArtifactItem failed on deletion count didn't reset properly", which is clearly unrelated to my pull request.
